### ISSUE TITLE
Forgot Password does not work (SSL Error)

### ISSUE THREAD

**Issue by sscholl** [RocketChat/Rocket.Chat#6943]
Rocket.Chat Version: 0.55.1
Running Instances: 1
DB Replicaset OpLog: Enabled
Node Version: 4.8.1 - x64

Hello,
my users reported, that the forgot password function is not working. They get the error message 
```Error trying to send email: Mail from command failed - 501```. The following error message is printed into the logs. No other issues are found. 

```
Error: 139676799047488:error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number:../deps/openssl/openssl/ssl/s3_pkt.c:365:
```

I use rocket chat inside a docker and this container is behind a proxy. Maybe this generates the issue.
Proxy Config is:
```
# REQUIRES:  a2enmod proxy
# REQUIRES:  a2enmod proxy_wstunnel
        <Location />
            Require all granted
            Options -Indexes
        </Location>

        ProxyPass /websocket ws://localhost:3000/websocket
        ProxyPassMatch ^/sockjs/(.*)/websocket ws://localhost:3000/sockjs/$1/websocket
        ProxyRequests Off
        ProxyPass / http://127.0.0.1:3000/
        ProxyPassReverse / http://127.0.0.1:3000/
```

For SMTP server, I use port 465 and the test mail is working.

Related:
https://github.com/nodejs/node-v0.x-archive/issues/6779

**Comment by sscholl** [comment]
I tested the docker container without a reverse proxy in between. The same error occures. This means the proxy is not the reason for this error.

**Comment by sscholl** [comment]
closed via #6940 in 0.56.0

Go to Administation->E-Mail->SMTP->Protocol (smtps)
